### ISSUE TITLE
An exception occurred when using WpfPlotGL: The GLWpfControl assembly could not be found

### ISSUE THREAD

**Issue by shijianoo** [ScottPlot/ScottPlot#4607]
**ScottPlot Version:5.0.47

**Question:Why does an error occur when I use the WpfPlotGL control? The error occurs in the second line (when adding the control to the Grid), which probably means that the GLWpfControl assembly cannot be found. I tried to uninstall "ScottPlot.WPF" using NuGet and then reinstall it, and even restarted VS, but it still didn't work. The GLWpfControl.dll file also exists in the bin directory. Creating it in code will give this error, but creating it in Xaml will still give the same error.This error occurred when I upgraded to version 5.0.47.  I don't remember the previous version, but I had been using WpfPlotGL.  Others can use it normally.  I don't know if there is something wrong with my project.

**in code
![Snipaste_2024-12-12_11-18-13](https://github.com/user-attachments/assets/abff31a8-db90-46af-aa9e-3c79472c8359)
**Code Sample:
```cs
 scottPlot = new WpfPlotGL();
 rootGrid.Children.Add(scottPlot);  //Error
 scottPlot.Margin = new Thickness(0, 10, 10, 0);
 Grid.SetRow(scottPlot, 2);
```

in Xaml
![Snipaste_2024-12-12_11-24-41](https://github.com/user-attachments/assets/49c822de-3de5-4adb-983b-0d6eb0339722)

** Exception
```cs
System.IO.FileLoadException: "Could not load file or assembly 'GLWpfControl, Version=3.3.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strong named assembly is required. (Exception from HRESULT: 0x80131044)"
```

**Comment by StendProg** [comment]
HI @shijianoo,

It looks like you are using `.Net Framework`.
It does have this error because it uses the outdated `GLWPfControl`. At the same time, the `.NET` version works fine in my tests.

I did a little research on the cause of this bug, it is related to this change in version `5.0.40` #4295.
If you disable assembly signing, the error you cited disappears. 
But `WPFControlGL` still does not work correctly, it draws a white screen. This is due to another bug introduced in version 4.0.36 #3976.
It seems that disabling anti-aliasing when drawing lines causes white screen.

All these errors apply only to the `.NET Framework` version which uses the outdated WPFControlGL version 3.3.0.

To summarize, I can give the following advice:
1. Switch to using `.NET`.
2. Stay with `ScottPlot 5.0.35` if you can't abandon the `.NET Framework`.
3. Wait for patches (if there are any) or participate in their development.

**Comment by shijianoo** [comment]
Okay, I'll wait.

**Comment by swharden** [comment]
Thanks for your insightful comments StendProg!

I'll add extend the list to add another option:

  4. Use the regular plot control (not the OpenGL one). For most applications the regular plot control offers excellent performance, and it is less prone to issues like the one described here.
